tur/seledroid-app/build.sh

```diff
@@ -0,0 +1,52 @@
+
+TERMUX_PKG_HOMEPAGE=https://github.com/luanon404/android-webdrivers
+TERMUX_PKG_DESCRIPTION="Chromium-based android webdriver application used by seledroid."
+TERMUX_PKG_LICENSE="Unlicense"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+TERMUX_PKG_VERSION=1.1.1
+TERMUX_PKG_SRCURL=https://github.com/luanon404/android-webdrivers/archive/refs/heads/main.zip
+TERMUX_PKG_SHA256=7d59ce35a9b86429b3ed36f5f6f46c92f49d3851d6f441beb9a7970449ccf2f8
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make_install() {
+    SHARED=$TERMUX_PREFIX/share/seledroid-app
+    mkdir -p $SHARED
+    cp Apk-Android-WebDrivers/Chromium/Chromium-$TERMUX_PKG_VERSION.apk $SHARED
+    echo $TERMUX_PKG_VERSION > VERSION
+    cp VERSION $SHARED
+    # Highly compressed install-seledroid-app script, ";" are replaced by "\n" later.
+    # compressed because so saves space in this build.sh script.
+    # because the install-seledroid-app script is short, it not have to be hosted, it can be here locally.
+    echo '#!/data/data/com.termux/files/usr/bin/bash
```

Comment:
I think these code should be store in a seperate file and copy this file to the target folder in `build.sh`. It is not readable for maintainers.

Reply:
The thing is that i don't know how to do that, i believed that it is not possible, but now i learned how, i already pushed the commit with the changes